dptb/tests/test_postprocess_hr.py

```
@@ -0,0 +1,45 @@
+        self.struc_path = os.path.join(self.root, "tests/data/e3_band/data/Si64.vasp")
+
+    def test_get_hR_real_model(self):
+        # Initialize System with real model
+        system = TBSystem(
+            data=self.struc_path,
+            calculator=self.model_path,
+            device='cpu'
+        )
+        
+        # Call get_hR
+        h, s = system.get_hR()
```

Ruff (0.15.0)

[warning] Unpacked variable `h` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)